### DISCUSSION TITLE
Import heic background from macOS

### DISCUSSION THREAD

**Discussion by Bellisario** [t1m0thyj/WinDynamicDesktop#332] [Q&A] [answered]
Is there a way to import dynamic macOS heic wallpapers directly in the app?

**Comment by ghost** [comment]
Depends. If it is a system HEIC wallpaper, it should be in WDD too. But if not, try searching for it on https://dynamicwallpaper.club/

**Reply by Bellisario** [reply]
Sorry, a little question: I've tried with https://dynamicwallpaper.club/, too, but it doesn't work equally... the file download it's in HEIC format. :disappointed:

**Comment by Bellisario** [comment, marked as answer]
Solved with [this comment](https://github.com/t1m0thyj/WinDynamicDesktop/issues/331#issuecomment-856978663).
